realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -1087,6 +2108,107 @@ public static void insertOrUpdate(Realm realm, Iterator<? extends RealmModel> ob
                 unmanagedcolumnRealmListList.add(item);
             }
         }
+
+        do {
+            RealmList<java.lang.String> managedcolumnStringListList = realmSource.realmGet$columnStringList();
```

Comment:
`managedcolumnStringListList` -> `managedColumnStringListList` upper case **C**

same for `unmanagedcolumnStringListList`

Reply:
hey, what about having a `RealmList.copyFromRealm()`? then we can simply code here a lot.

I am fine with this as well. it is minor.

Reply:
I simplified the code.